postman/patient-demographics-sandbox.postman_collection.json

```diff
@@ -907,28 +1295,52 @@
 									"value": "eq2010-10-22"
 								},
 								{
-									"key": "phone",
-									"value": "01632960587"
+									"key": "_max-results",
+									"value": "1"
 								}
 							]
 						},
-						"description": "Example of a wildcarded search, where the \"family\" name is wildcarded, when phone is included will return 1 patient - Jane Smith patient (nhsId - 9000000009)"
+						"description": "Example of a wildcarded search with limited results, where the \"family\" name is wildcarded and \"_max-results\" is added, which limits the number of results to 2 patients - will return a result of 2 patients - Jane Smith patient (nhsId - 9000000009) & Jayne Smith (nhsId - 9000000017)\r\n\r\nNote: if \"_max-results\" is reduced to 1 then a \"TOO_MANY_RESULTS\" error response will be returned, as the number of matched patients found is greater than the \"_max-results\" parameter."
```

Comment:
Duplication of wording: "which limits the number of results to 2 patients - will return a result of 2 patients"

Reply:
looks fine to me